platform/linux/instructions.txt

```diff
@@ -0,0 +1,17 @@
+KOReader %%VERSION%% (Linux %%ARCH%%)
+--------
+
+## Requires
+
+- `GLIBC` >= %%ABI%%
+- `SDL2`
+
+## users
+
+#### You shouldn't use this package unless you know what you're doing. Please use [AppImage](https://github.com/koreader/koreader/wiki/Installation-on-desktop-linux#what-is-an-appimage) instead.
+
+## package maintainers
+
+The script in **`bin/`** expects the rest of the **assets** in **`../lib/koreader`**. This works with **`/usr`** and **`/usr/local`** installations. If you want to install elsewhere make sure to tweak the script.
```

Comment:
How come it can't be installed elsewhere?

Reply:
Do you mean, install in `/opt/koreader`? and symlink /opt/koreader/bin/koreader to somewhere in PATH. That should work, yeah.

I was thinking about putting everything in /opt except the script. That would require a little change.

If you feel the info here is redundant and package maintainers should be able to figure those things on their own I'm happy to remove it all :)

Reply:
> Do you mean, install in `/opt/koreader`?

I meant that it should be relocatable, as long as the tree structure is kept.

> and symlink /opt/koreader/bin/koreader to somewhere in PATH. That should work, yeah.

It won't, it's missing a realpath/readlink to get the path to the real launcher.

> I was thinking about putting everything in /opt except the script. That would require a little change.

It would make things simpler: put everything in the same directory.

> If you feel the info here is redundant and package maintainers should be able to figure those things on their own I'm happy to remove it all :)

I just think the last sentence is wrong and can be removed.